hole/quadratic-formula.go

```diff
@@ -0,0 +1,142 @@
+package hole
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"strings"
+)
+
+const DEBUG = false
+
+type solution struct {
```

Comment:
This is a very generic name for something that is scoped across all holes, maybe call it `quadraticSolution` at least.